libmambapy/src/libmambapy/bindings/solver.cpp

```diff
@@ -302,36 +302,44 @@ namespace mambapy
             ))
             .def_readwrite("actions", &Solution::actions)
             .def(
-                "to_install",
+                "packages",
                 [](const Solution& solution) -> std::vector<specs::PackageInfo>
                 {
+                    // TODO(C++23): std::ranges::to
                     auto out = std::vector<specs::PackageInfo>{};
-                    out.reserve(solution.actions.size());  // Upper bound
-                    for_each_to_install(
-                        solution.actions,
-                        [&](const auto& pkg) { out.push_back(pkg); }
-                    );
+                    out.reserve(solution.actions.size());  // Lower bound
+                    for (const auto& pkg : solution.packages())
+                    {
+                        out.push_back(pkg);
+                    }
                     return out;
                 }
             )
+            .def(
+                "to_install",
+                [](const Solution& solution) -> std::vector<specs::PackageInfo>
+                {
+                    // TODO(C++23): std::ranges::to
+                    auto range = solution.packages_to_install();
+                    return { range.begin(), range.end() };
```

Comment:
Doesnt `{` calls the consrtuctor which takes only values? 
Or is this callilng the range constructor because the type doesnt match the values?
I assumed that you would get compilation errors in that case, not overload resolution.

Reply:
OK I checked and that works as expected. TIL